backbone/src/main/java/org/researchstack/backbone/answerformat/TextAnswerFormat.java

```diff
@@ -1,16 +1,21 @@
 package org.researchstack.backbone.answerformat;
 
 
+import android.text.InputType;
+
 /**
  * The TextAnswerFormat class represents the answer format for questions that collect a text
  * response from the user.
  */
 public class TextAnswerFormat extends AnswerFormat
 {
     public static final int UNLIMITED_LENGTH = 0;
-    private int maximumLength;
+    int maximumLength;
+    int minimumLength = 0;
 
-    private boolean isMultipleLines = false;
+    boolean isMultipleLines = false;
+    int     inputType       = InputType.TYPE_CLASS_TEXT;
+    String  validationRegex = null;
```

Comment:
That's a good question. My vote is to stick with validationRegex to keep parity with iOS.